src/assets/styles/articles.scss

```diff
@@ -112,3 +112,23 @@
   font-style: bold !important;
   font-weight: 900;
 }
+
+.socialShare{
+  margin-top: $onepercent;
```

Comment:
Kindly use the exact values instead of variables as of this case or rather why did you opt to store the value and use the variable names?

Reply:
I opted to variables to avoid repetition of the same values more than twice.

Reply:
Done.